docs_user/modules/con_changes-to-cephFS-via-NFS.adoc

```diff
@@ -2,14 +2,14 @@
+* On {OpenStackShort} {rhos_prev_ver}, Pacemaker controls the high availability of the `ceph-nfs` service. This service is assigned a Virtual IP (VIP) address that is also managed by Pacemaker. The VIP is typically created on an isolated `StorageNFS` network. There are ordering and collocation constraints established between this VIP, `ceph-nfs`, and the {rhos_component_storage_file_first_ref} share manager service on the Controller nodes. Prior to adopting {rhos_component_storage_file}, you must adjust the Pacemaker ordering and collocation constraints to separate the share manager service. This establishes `ceph-nfs` with its VIP as an isolated, standalone NFS service that you can decommission at will after completing the {rhos_acro} adoption.
 
-* In {Ceph} 7.0, a native clustered Ceph NFS service has to be deployed on the {Ceph} cluster by using the Ceph orchestrator prior to adopting the {rhos_component_storage_file}. This NFS service eventually replaces the standalone NFS service from {OpenStackShort} {rhos_prev_ver} in your deployment. When the {rhos_component_storage_file} is adopted into the {rhos_acro} {rhos_curr_ver} environment, it establishes all the existing exports and client restrictions on the new clustered Ceph NFS service. Clients can continue to read and write data on their existing NFS shares, and are not affected until the old standalone NFS service is decommissioned. After the service is decommissioned, clients can re-mount the same share from the new clustered Ceph NFS service during a scheduled downtime.
+* In {Ceph} {CephVernum}, a native clustered Ceph NFS service has to be deployed on the {Ceph} cluster by using the Ceph Orchestrator prior to adopting the {rhos_component_storage_file}. This NFS service eventually replaces the standalone NFS service from {OpenStackShort} {rhos_prev_ver} in your deployment. When the {rhos_component_storage_file} is adopted into the {rhos_acro} {rhos_curr_ver} environment, it establishes all the existing exports and client restrictions on the new clustered Ceph NFS service. Clients can continue to read and write data on their existing NFS shares, and are not affected until the old standalone NFS service is decommissioned. After the service is decommissioned, clients can re-mount the same share from the new clustered Ceph NFS service during a scheduled downtime.
```

Comment:
In the bullet points in L11 and L13, does “client” refer to the human user or a computer?

Reply:
Hi @klgill thank you for your changes and they look good to me.
Regarding `clients` on L11, I originally intended it as the "software that uses the NFS Share". Clearly, the re-mount operations is not something automated, and the ability to do that depends on the workload, so it is still an operation (the re-mount) that should be performed by a human administrator. 
Does this properly answer your question?

Reply:
Yes, it does. Thanks! I'll likely change "client" to "you", since that's our style when referring to the human user.